vta/tutorials/frontend/legacy/deploy_detection.py

```diff
@@ -58,7 +58,7 @@
 from tvm import rpc, autotvm, relay
 from tvm.relay.testing import yolo_detection, darknet
 from tvm.relay.testing.darknet import __darknetffi__
-from tvm.contrib import graph_runtime, graph_runtime, util
+from tvm.contrib import graph_runtime, graph_runtime, utils
```

Comment:
Do you mind remove the duplicated `graph_runtime` here as well?

Reply:
Duplicated import is removed.